extras/mobile-view.rst

```diff
@@ -0,0 +1,156 @@
+
+   * - .. figure:: /images/extras/mobile-view/mobile-view-login.png
+          :alt: Mobile View Login Screen
+          :align: center
+          :width: 100%
+
+          Login Screen
+
+     - .. figure:: /images/extras/mobile-view/mobile-view-home.png
+          :alt: Mobile View Home Page
+          :align: center
+          :width: 100%
+
+          Home Page
+
+     - .. figure:: /images/extras/mobile-view/mobile-view-ticket-create.png
+          :alt: Mobile View Ticket Create Screen
+          :align: center
+          :width: 100%
+
+          Ticket Create Screen
+
+   * - .. figure:: /images/extras/mobile-view/mobile-view-ticket-detail-view.png
+          :alt: Mobile View Ticket Detail View
+          :align: center
+          :width: 100%
+
+          Ticket Detail View
+
+     - .. figure:: /images/extras/mobile-view/mobile-view-article-reply.png
+          :alt: Mobile View Article Reply Dialog
+          :align: center
+          :width: 100%
+
+          Article Reply Dialog
+
+     - .. figure:: /images/extras/mobile-view/mobile-view-organization.png
+          :alt: Mobile View Ticket Organization Tab
+          :align: center
+          :width: 100%
+
+          Ticket Organization Tab
+
+.. hint::
+
+   We intentionally do not provide specific instructions and comprehensive
+   documentation for the mobile view! The overall UX should be intuitive and
+   self-explanatory in most cases.
+
+Features
+--------
+
+Mobile view provides you with a way to do your common Zammad daily tasks while
+on the go:
+
+   * Manage & use your ticket overviews
+   * Search for existing records
+   * Create a new ticket
+   * Reply in an already existing ticket
+   * Modify ticket attributes
+   * Modify customer attributes
+   * Modify organization attributes
+
+Mobile view also has some exclusive features:
+
+   * Innovative Zammad UX
+   * Next-level accessibility
+   * Ready for multi-device usage
+   * Progressive web app (PWA) support
+
+Limitations
+-----------
+
+Mobile view is also currently missing some features which are provided by the
+desktop view:
+
+   * Ticket Article Time Accounting
+   * Ticket Article **Split** Action
+   * Linked Tickets & Ticket **Link** Action
+   * Ticket Macros
+   * Ticket History
+   * Ticket Create Templates & Shared Drafts
+
+Additionally, certain features were intentionally omitted in order to improve
+the focus on important information:
+
+   * Most Management Features (except ticket user and organization management)
+   * Most Knowledge Base Features (except ticket integration)
+   * Most User Profile Functions (except avatar and language preferences)
+   * Reports
+   * Caller Log
+   * Live Chat
+
+Access
+------
+
+Zammad now implements a mobile device detection, which results in automatic
+redirection to mobile view. Even with this mechanism in place it's possible to
+explicitly switch between the views by using app links:
+
+   * **Continue to desktop** link in mobile view
```

Comment:
I haven't found a "better" solution for it.
I personally don't like listings of that kind but that's my taste

![image](https://user-images.githubusercontent.com/6549061/233394272-e4b68917-9ea3-46cb-84b3-0eb73a6eab97.png

By using 
```
**Continue to mobile** link in desktop view
   .. container:: cfloat-left

      .. figure:: /images/extras/mobile-view/mobile-view-desktop-login-continue-to-mobile.png
...
```

The whole heading is written strong without any listing flair. Maybe an option.

![image](https://user-images.githubusercontent.com/6549061/233394983-493e774e-e3ae-4aef-8065-5d5c5dee95b6.png)